### PR TITLE
chore(demo): move version manager to header

### LINKED ISSUE #3709: 📚 - Documentation redesign
### What is the affected URL?

https://taiga-ui.dev

### Description

https://www.figma.com/file/e7haZlhOm9DjupCkq5UIMe/Taiga-3.0?node-id=31602%3A136510&t=6zYP6BtXIzsghQbH-11

image

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
Close #3709

image
image

### CODE REVIEW

File: projects/demo/src/modules/app/version-manager/version-manager.component.ts [hunk 1]
```diff
@@ -15,15 +15,7 @@ import {TUI_VERSIONS_META_OPTIONS} from './versions.constants';
...
-    imports: [
-        FormsModule,
-        NgForOf,
-        NgIf,
-        TuiDataList,
-        TuiSelectModule,
-        TuiStringifyContentPipe,
-        TuiStringifyPipe,
-    ],
+    imports: [FormsModule, NgForOf, TuiButton, TuiChevron, TuiDataList, TuiSelectModule],
```
Comment (waterplea) [inline projects/demo/src/modules/app/version-manager/version-manager.component.ts:18]: Looks like we do not need forms and select here

File: projects/demo/src/modules/app/version-manager/version-manager.template.html [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-    [valueContent]="'label' | tuiStringify | tuiStringifyContent"
+<button
+    appearance="whiteblock"
+    size="s"
+    tuiButton
+    tuiChevron
+    type="button"
+    [tuiDropdown]="content"
+    [tuiDropdownLimitWidth]="'fixed'"
+    [tuiDropdownMaxHeight]="700"
```
Comment (waterplea) [inline projects/demo/src/modules/app/version-manager/version-manager.template.html:9]: We don't need this one

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/app/version-manager/version-manager.component.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/app/version-manager/version-manager.component.ts]: fixed

File: projects/demo/src/modules/app/version-manager/version-manager.template.html (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/app/version-manager/version-manager.template.html]: fixed

**Review by mdlufy — APPROVED** [review]

**Pull request merged**